docs/scu/dev/features/integrating-code-releases.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Integrating code releases
+description: Spryker Code Upgrader automatically integrates code releases
+template: concept-topic-template
+---
+
+Spryker architecture supports the extensions of modules using extension points in the project code. With Spryker Code Upgrader, the process of integrating plugins, configuration updates, adding new translations, and other code changes is automated. As a result, to add new features, developers don't need to apply code changes manually.
+
+The Upgrader supports the following extension points:
+- Configuration of the following:
+  - Module
+  - Environment
+- Copying module files
+- Adding and removing the following:
+  - Plugin
+  - Navigation element
+  - Widget
+  - Glue relationship
+- Adding the following:
+  - Transfer definition
+  - DB schema definition
+
+## Release integrability rating
+
+Release integrability rating is a measure of the integrability coverage of a code release. It is the percentage of automatically integratable plugins, configurations, schemas, and other components.
+
+This rating is determined during the internal testing where the generated integration code is compared with the manually released code to assess the level of integration achieved. If both are equal, the release is fully auto-integratable.
+
+There are types of releases that provide only dependency updates without any code integrations, so-called module-only releases. Such releases are 100% auto-integratable.
+
+## Release integrability rating threshold
+
+By default, the Upgrader creates PRs for all the releases with rating 70% or more. The description of the PRs contains the rating of the suggested release.
```

Comment:
What if I integrate a 70% release? Isn't there any manual development required for the remaining 30%?

Reply:
70% it's a threshold - min release rating for automatic integration. If release has less than 70% rating, upgrader just updates the dependencies, the automatic integration will be skipped (must be provided manually)